pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
```

Comment:
We could add this to `tox.ini` and `setup.cfg` to not add even more files see https://docs.pytest.org/en/latest/customize.html#finding-the-rootdir

Reply:
Would probably favor `setup.cfg`. Note that this line then becomes `[tool:pytest]`